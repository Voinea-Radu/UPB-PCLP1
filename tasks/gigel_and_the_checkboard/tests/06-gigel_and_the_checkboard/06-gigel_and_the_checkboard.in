64
-425 270 343 338 355 231 430 409 -106 216 -188 11 -285 -280 175 -288 -118 349 -122 258 -79 288 -340 359 -452 121 250 272 165 371 342 -498 -204 -467 -421 421 358 -113 -431 101 -319 -315 163 -139 -63 397 34 -274 439 256 160 -104 24 128 -63 96 -493 -187 -449 406 130 227 180 487 
-63 308 -107 -161 30 192 -317 -256 -46 -143 359 -402 13 15 92 213 -142 -21 352 125 466 191 461 146 -120 263 -183 126 -175 483 -491 478 414 162 -311 235 -330 -326 374 -324 56 -360 -483 -419 21 -91 12 -250 -496 313 -292 69 -367 -46 113 338 -467 -164 294 156 -320 -23 380 -496 
-83 -402 -1 -54 -17 465 -273 150 290 -377 116 -127 -138 -460 396 -200 -339 -372 -168 -411 369 -259 -245 292 -280 280 -278 -42 14 78 408 -382 -444 377 322 8 -306 -287 -4 365 -121 -15 139 330 23 318 243 -361 387 277 484 427 401 -343 -242 -357 156 -192 491 52 404 506 -474 -8 
89 -234 -214 -168 242 307 359 372 55 -408 345 344 4 452 329 -116 -89 -471 42 -115 -278 -414 216 15 157 394 -377 89 373 502 112 334 -279 318 65 501 -191 -320 -497 -37 28 384 334 81 223 302 -283 -369 -306 -350 494 286 -350 292 -102 306 249 508 90 332 -158 324 -474 -294 
-155 81 -108 -179 324 25 -324 160 262 364 -349 247 -361 -256 256 -348 -288 272 474 173 -396 -504 88 130 -238 -2 193 -66 -247 131 -28 334 -501 425 -409 444 18 246 -307 234 -159 96 311 -38 -490 -319 204 -301 -17 -454 309 271 -3 -141 195 -250 35 371 -228 52 19 331 -283 -1 
99 471 -247 216 -383 -174 -435 -188 -438 11 369 -341 44 198 283 -478 204 64 -73 -340 510 343 253 391 421 -106 287 452 292 -239 181 477 332 222 351 -186 158 453 -165 506 473 -460 -368 -340 -12 -363 402 -27 161 -480 476 -247 -202 -177 333 159 -196 -222 -210 -294 194 -250 -310 -270 
112 -167 -50 455 -161 485 75 -252 -145 471 173 135 24 -441 -193 -311 296 25 280 425 -248 307 -79 88 253 -417 -16 426 -166 374 471 105 -81 -310 432 -41 293 389 -144 133 -287 467 59 48 283 332 -511 373 -73 55 209 153 -250 133 473 212 185 352 -66 -459 -92 99 -274 -66 
-186 427 -264 140 -469 427 362 69 115 82 384 406 459 170 -128 471 349 96 -156 106 -433 91 139 125 -22 -136 -263 -502 -34 9 189 -247 -55 128 386 305 87 -119 272 268 -370 -220 -111 -328 7 -146 -36 -487 110 373 -84 477 174 465 -141 189 -403 469 493 -67 56 349 -199 -145 
36 -431 32 -119 -178 -198 -313 -473 350 -275 17 445 -303 -459 9 478 141 107 -483 -307 -406 -258 -127 -47 -1 -71 -447 -452 512 -168 -98 95 -271 -436 458 -130 241 203 -495 32 294 -484 -379 420 -362 129 -71 54 -29 -130 475 441 -362 457 263 468 -79 -405 -74 465 234 275 -458 -4 
-461 -262 124 -36 182 264 -193 76 -34 204 414 -77 -181 345 -508 176 129 -361 -169 -13 45 -44 279 455 -107 -460 462 -335 435 -431 -47 -509 219 -91 486 -171 -456 -223 -449 291 -221 -309 -296 -366 488 57 193 344 287 309 169 24 -462 -42 465 67 422 -175 459 -294 -141 -424 289 -219 
-457 -473 -283 378 -164 435 -347 425 233 -102 195 -154 -318 -292 431 477 -168 -388 227 -323 148 -102 -378 -323 336 453 491 -233 361 201 64 464 475 469 -385 -483 38 295 494 189 116 -467 69 169 -416 194 -369 -174 -407 59 416 -99 -231 -164 -122 325 300 176 -388 -389 158 272 131 282 
170 -114 -512 -424 -117 -6 358 27 76 -501 -340 55 15 -53 -349 -251 136 342 -182 -93 239 -111 -426 -172 72 -142 462 -93 181 -168 -143 -34 -308 -257 -233 -391 -409 402 243 137 163 451 42 433 176 -53 -56 129 -406 -101 177 27 -252 -162 -445 235 36 -224 169 366 -456 -217 -338 -474 
430 47 226 -259 131 -201 444 49 35 -448 52 119 -265 -290 438 -496 -85 -477 57 -325 -120 -334 -502 -18 -227 319 -454 -17 110 -293 251 495 24 207 268 -169 -127 -492 30 134 -172 -489 101 -348 149 356 169 4 271 188 442 -420 -145 177 222 232 164 508 512 -157 -108 -25 -12 113 
-488 -309 192 -200 -204 64 -183 -497 -262 156 428 302 -122 109 -80 475 -34 -447 69 -69 -200 -239 210 -142 -269 -104 -185 218 253 -60 33 173 -133 -375 119 -486 -200 14 220 -126 208 -459 87 466 -311 -306 188 -161 426 341 -163 -55 50 -303 355 -279 -396 331 359 132 -349 -294 -375 -47 
-288 154 135 -269 -63 436 -436 295 -154 -147 -438 -452 188 -78 251 -162 201 365 -352 -114 -89 -138 492 -355 444 -238 -165 222 17 139 287 333 -102 232 -294 241 224 219 -85 88 -18 -373 130 41 -165 410 483 -487 46 271 44 -181 -133 -215 -165 -453 467 -176 57 -1 354 456 83 440 
-280 -301 149 -167 -144 504 -426 112 351 -335 127 -479 -336 -418 -7 -482 -66 331 314 139 76 -215 -508 -334 263 24 -232 224 309 -512 223 -370 316 -63 -219 -278 176 208 180 -54 -400 405 5 403 101 -57 -413 95 -467 -460 -161 156 -480 285 -355 -200 -483 173 474 302 450 390 -352 -219 
36 -316 77 -48 -197 -277 52 -434 -133 -338 226 -492 -208 487 -251 -401 85 -373 -108 -208 58 313 -434 -95 86 243 431 267 -237 416 431 -346 377 -239 -47 210 282 -309 -240 -330 259 -236 -439 -14 -509 -461 21 -288 -492 73 -242 56 -401 -148 512 -302 8 122 -379 194 222 -224 -453 104 
396 -108 259 222 157 -95 27 -476 -282 345 84 -11 -316 -133 494 445 -434 296 511 -435 494 100 -497 94 79 -233 488 70 123 467 410 -310 319 -409 -433 471 -186 98 109 480 224 335 101 -254 -144 -306 208 396 396 -58 -1 -297 336 -4 -106 77 -478 -147 489 453 -299 -496 142 244 
-378 241 -274 468 -512 376 -418 125 -471 405 -428 -255 -242 -393 -170 -217 -404 236 306 -468 -201 172 -307 10 250 -440 -301 -354 -273 385 468 159 -202 -23 -207 -72 -392 -271 -203 -227 -112 186 -73 225 23 199 -203 -488 449 -325 -36 193 -369 -325 -219 220 -194 -209 -504 306 428 -144 -495 86 
492 -479 378 -12 -116 -35 -410 -381 -94 -242 -154 340 457 -360 350 -406 367 244 109 -110 -386 -404 -495 250 -343 -146 283 -375 500 -334 -288 433 475 -330 311 -334 -312 121 188 262 500 -37 394 257 34 502 149 298 17 -92 82 -99 56 -58 274 -386 -190 85 -46 -338 380 316 83 461 
41 -433 -268 235 -202 228 -457 -252 429 226 -123 508 -422 511 -133 442 -455 201 -506 90 465 109 276 -236 -81 -457 346 327 -246 -37 -179 -98 -446 460 191 221 283 -406 193 501 363 -332 -426 -140 283 224 118 18 -314 100 452 321 -82 5 -32 -100 79 319 207 468 -275 400 -98 93 
139 -336 -354 -440 166 295 120 465 352 352 -395 504 454 -280 229 29 -5 -310 -482 -237 -220 162 331 492 341 -396 -120 -84 303 99 131 -142 164 -445 -52 -26 202 -323 112 11 -297 -404 -146 -487 -115 125 -246 -430 -501 -21 52 -54 208 -42 437 -284 172 65 -230 -67 80 -297 121 349 
-331 -132 -406 -1 -490 467 -233 -6 162 -361 212 -173 -295 -239 -49 258 -325 -507 175 361 -231 260 101 303 50 342 378 122 288 503 385 1 391 18 -28 310 242 133 222 -235 505 -459 312 103 250 -36 -192 227 441 -83 -55 -379 -137 -472 -81 -101 331 -440 -477 -361 -232 2 -376 -289 
-189 426 -91 145 -211 469 -456 388 -373 70 140 288 -152 256 342 193 -400 -319 90 -191 302 -240 -279 -453 -237 67 -82 -423 353 -429 -5 -440 -479 -14 -472 465 346 488 468 -102 -453 -403 -312 -298 -111 -250 -52 11 28 -493 -45 -47 -461 462 406 -209 475 -444 -187 -412 -353 510 -401 391 
-179 263 -222 -157 -284 -38 -218 500 -120 424 -185 115 -68 -371 -20 301 -67 -46 343 -419 410 -510 -211 -437 -489 194 315 -173 266 -436 -175 -77 -500 234 -39 -233 -18 163 -95 390 -255 -295 165 -305 73 267 -139 198 393 -166 -262 20 449 -235 290 164 398 342 330 414 -236 399 195 -286 
-248 -189 -351 354 -348 -7 -29 -485 -317 -486 261 408 362 144 70 -254 -92 -304 74 216 322 434 -363 -93 -138 -303 326 413 -487 148 288 264 -487 -50 -167 -429 117 -201 203 411 -185 80 33 -234 -275 131 -440 200 398 382 336 55 128 301 -287 282 -80 -115 -478 249 -400 386 339 5 
-25 420 145 -370 6 -308 238 -154 -229 -452 301 264 298 226 -340 396 448 -385 -403 -510 214 -402 -434 90 100 -109 -351 482 109 378 506 449 -86 -512 90 332 -510 -374 -362 165 38 -212 297 -211 154 235 -189 263 211 -345 -43 -89 -197 -296 -259 202 128 -390 509 -298 218 -510 147 153 
-451 -67 485 -455 119 -394 -74 409 -299 -510 306 37 -392 368 -467 488 39 508 -409 -100 -79 187 172 440 -146 105 -114 -398 -98 -236 512 485 1 -256 -378 23 -346 278 417 437 -266 -67 119 31 201 287 242 239 -496 -432 -136 79 131 119 306 -487 243 360 68 271 -392 309 214 148 
-300 -356 306 177 103 -250 -43 38 -437 -457 -273 -227 474 -501 265 -448 -395 -271 381 214 -151 -47 -116 -185 226 -359 126 68 -169 263 187 -5 -429 -176 -95 -82 76 441 302 329 263 -270 -90 44 251 -494 -347 193 -233 -215 -256 -382 90 450 493 -168 125 -284 -443 200 -508 -412 -361 111 
-194 -451 -318 440 295 504 -47 388 494 -165 -174 -308 -143 -227 6 -430 -482 -103 -182 178 50 400 -505 -409 64 111 366 -237 96 197 202 472 -88 428 -145 -107 367 -413 -102 -379 404 -386 -492 142 12 106 99 290 -377 -183 79 -106 -510 78 63 6 99 -227 -52 437 -377 -362 -127 274 
-57 -8 -147 414 -113 -262 111 -245 339 426 332 -38 -111 -512 323 149 126 -259 -242 210 123 -512 -302 196 -448 -254 499 -253 -232 473 -487 -457 -457 483 331 335 -108 426 270 -437 -346 492 407 -83 107 -384 20 497 -45 406 473 215 169 -181 -425 -427 -309 -497 -481 146 -187 -177 -431 -316 
-218 19 -68 -146 -94 -223 -461 270 -139 339 -408 215 297 -28 166 -172 431 232 206 12 377 -196 205 276 446 -298 160 333 327 -72 -410 -192 290 -133 222 394 -474 391 166 -53 -49 -355 390 374 -11 -230 437 -118 385 448 -69 450 -431 336 -85 26 -78 207 -273 421 -162 -386 -301 312 
-180 190 191 20 357 -46 377 -464 -209 -80 139 240 89 356 -428 -98 346 -241 194 422 366 -268 -169 -23 -480 -166 -422 -467 -187 188 268 -46 128 -400 387 -509 -419 10 381 -292 21 -275 -199 124 -497 126 92 141 113 318 -184 13 -72 136 -140 450 -109 361 418 30 452 -440 -89 196 
-442 282 -254 -154 -488 -232 216 183 -256 -332 -471 236 -98 88 -171 57 -465 -160 -425 -358 -165 -453 -182 13 354 498 185 167 506 -233 -166 -493 354 2 -476 -289 54 400 -60 258 232 -363 244 -240 279 -172 -39 -59 387 376 -88 171 104 69 505 313 70 498 374 442 -297 -431 334 500 
-452 168 416 236 164 -347 37 460 369 -424 -22 -161 441 134 72 52 -11 469 410 21 -408 -246 -52 228 -443 -68 -60 421 -306 228 131 322 -128 -293 422 -196 48 308 -11 -139 -448 452 -456 323 -395 360 -170 144 491 -370 418 126 487 -76 131 213 485 387 -508 -510 307 -475 -138 -431 
-126 112 225 -344 489 -95 374 119 -301 -456 280 -452 -71 -391 424 461 5 12 255 84 -21 -329 -423 345 346 -211 221 -40 232 251 176 167 -4 48 431 471 -131 149 393 240 -348 -393 -284 -509 399 -458 -379 471 474 -270 -361 -297 72 -342 315 -483 -308 0 -218 336 -301 64 47 315 
186 -382 -89 155 226 209 116 468 461 -143 217 -361 214 -309 17 218 492 18 66 223 442 184 -285 -412 62 510 -505 -464 -170 369 246 -454 215 138 -344 506 322 478 511 -421 -265 215 80 -439 33 149 -298 -138 -362 108 -272 -433 -54 -142 -335 -137 -341 426 -80 -233 504 311 207 -359 
47 134 -408 200 194 171 335 275 -495 -68 -27 260 -301 480 157 64 -505 -286 -85 -112 -26 141 445 357 49 341 -326 -123 376 -1 -280 93 -117 455 347 -428 278 201 398 -233 -500 509 -23 242 211 93 -324 -188 -182 92 38 -395 189 375 7 125 -373 188 -20 -443 438 451 -290 -50 
-301 194 277 225 388 -222 381 223 4 401 -98 193 -251 -103 -140 -476 29 329 -354 209 -200 271 -98 284 -8 273 -344 449 -237 91 -9 113 198 382 418 431 -409 -83 198 298 295 -121 485 -218 10 115 -26 -128 124 -189 312 413 -479 110 46 21 231 503 -379 502 68 -266 314 -282 
-305 467 -226 122 -91 -319 329 107 -11 -456 457 82 -380 388 -375 32 148 52 303 -326 356 48 45 -156 -203 -124 467 -456 247 -347 -457 409 -146 -17 451 -247 -104 -364 180 -483 28 -88 459 -348 160 316 97 238 7 -213 -496 -17 212 238 327 -255 510 479 -305 -486 380 -295 -487 -490 
251 -353 489 -209 349 -101 22 299 -161 -434 -32 439 215 215 -506 70 -317 499 242 149 -78 -460 -398 80 -430 263 218 -378 -250 439 464 -16 217 124 143 319 234 23 378 -35 -77 114 -143 -289 140 66 -479 288 -289 -257 105 14 345 469 -496 298 331 443 -461 -213 59 451 -493 38 
-407 -281 -73 271 -35 320 -91 -134 -498 -55 434 56 -324 -247 421 441 -150 420 465 -372 266 -294 364 336 191 141 -152 258 221 -211 -508 -435 -371 -500 -83 -25 104 -13 -342 130 52 103 -9 -183 336 77 141 349 241 309 -161 147 160 -194 -64 42 -511 205 -9 227 77 411 504 429 
506 -75 140 -496 -112 -270 430 436 -65 -377 425 -313 -458 87 -316 100 -29 333 -129 -336 315 -79 -51 -200 -101 -356 9 500 -1 -414 -170 229 -198 -29 266 411 -365 -178 241 247 -5 -78 -504 -153 -273 -494 314 385 -468 424 -8 -486 -332 58 -318 -484 446 327 360 -103 -84 307 -26 -92 
-205 -401 -265 303 -249 -232 -215 155 317 -317 438 -36 404 501 -329 8 209 24 -393 41 -48 -279 156 143 -452 474 -342 21 433 -433 193 -279 -399 -156 301 4 103 347 319 323 -133 -475 221 20 -387 -329 -468 441 -157 -13 -285 -295 -176 126 345 -300 -276 488 -106 -433 61 -175 -90 -430 
331 -104 449 -285 102 -487 -1 -22 136 76 -165 233 -283 112 428 174 -369 -35 46 -313 18 218 -68 -209 -196 411 -478 508 373 68 85 -364 -74 192 -31 23 338 67 279 31 28 -252 -134 261 442 116 415 421 264 303 314 -178 -483 276 -364 -139 477 -308 245 497 -297 369 319 258 
202 -168 -223 -6 -90 391 272 168 -133 475 505 -105 46 -302 37 176 -375 232 -237 -343 -240 120 492 -156 12 299 -468 -334 505 -122 387 -303 -200 -419 -9 133 -380 -251 18 -342 -59 450 -44 -333 -493 -412 508 368 174 207 155 -293 235 412 159 -137 -398 -253 -210 -334 385 -143 301 -145 
140 -258 -401 509 329 312 -506 -332 -380 -503 -87 197 342 -145 291 -250 -241 67 180 352 371 -166 -205 -304 -390 149 352 468 -99 -447 -342 -223 329 317 -486 500 9 -258 -173 185 165 178 97 420 -455 -48 226 68 377 490 -512 -163 -303 251 -388 70 -193 285 5 -88 214 -61 360 -95 
-434 -43 384 286 228 307 82 -418 309 381 29 59 -132 -485 -463 227 456 286 487 -447 422 -432 -103 4 167 213 268 422 -96 471 270 503 478 2 467 -158 -257 -55 -484 -134 -462 171 -117 -185 -263 225 -405 -216 310 114 116 -478 151 -445 -213 357 -407 200 272 -438 -261 -215 87 -395 
-299 197 -358 -120 367 5 76 -381 -117 11 -74 -370 -116 418 140 450 315 98 -16 -35 -304 -310 -302 462 153 -259 -168 179 233 352 443 20 87 -187 -63 350 -92 -467 455 293 -135 283 -50 71 -434 466 -405 456 -364 273 -74 355 -172 -439 134 -3 464 -393 413 -93 -458 -253 -72 -302 
-505 50 71 459 -324 -386 -126 344 -266 -44 43 -74 -414 169 380 -351 -210 -328 -122 245 370 -45 -354 408 -448 477 427 -82 -189 -424 39 -208 403 -262 390 29 -171 -212 176 168 131 -27 -361 -283 -70 -399 -418 -9 316 -64 3 258 174 -327 353 166 2 -444 336 280 -184 -257 184 248 
-394 -60 -46 -155 16 179 -509 -58 -353 -496 -285 -474 -69 -175 -337 289 -288 -188 -469 -43 -439 336 414 -136 56 -292 506 -360 273 -43 -413 -145 -510 491 -349 74 -3 -188 -278 281 381 310 -189 380 -370 204 -43 -109 -136 -366 -332 264 -90 73 -390 268 499 49 -195 -446 276 -163 7 37 
110 -60 215 273 126 256 24 -217 203 221 -47 -162 -402 -240 -342 78 -160 64 -352 393 -443 482 -215 -269 43 359 159 43 78 -453 447 331 -346 126 -449 266 -282 244 -231 -248 255 20 -234 267 -19 -323 -441 -209 -479 -67 380 122 396 358 282 -442 -299 -484 352 -257 -373 155 303 -404 
-292 120 -60 332 -226 -235 211 -374 40 -441 292 221 88 297 -34 56 413 250 -344 -395 -301 -69 330 -298 -267 -207 -418 -59 -245 -269 -264 -295 332 -447 486 231 -366 -74 180 430 52 354 487 195 248 389 -389 -372 479 -376 117 366 -103 92 86 391 228 170 362 -397 153 -398 -441 -190 
495 -250 -467 372 395 174 404 -90 492 -65 -303 -145 -319 203 -161 -369 465 72 488 34 209 -353 -230 186 -97 -250 -427 236 -169 -157 292 47 -206 -17 -81 -11 197 -263 -128 -247 -231 117 -186 -353 -465 -232 -48 -89 -138 159 -270 53 314 458 -354 6 148 346 137 36 215 -317 383 -100 
-72 25 381 -59 -269 233 387 108 509 10 -388 -122 83 -399 -84 -59 211 -109 -464 -91 -142 -62 -383 377 492 -306 198 -275 477 -80 -490 -154 100 -51 -198 432 424 -231 315 281 466 -293 -328 92 -206 217 -503 -64 424 328 348 -100 144 91 -7 -129 368 200 329 493 -26 -365 466 -254 
-351 469 -448 182 -11 -56 -51 148 -457 134 -120 -140 233 441 252 128 374 435 -458 314 -156 -339 272 -125 500 -384 204 499 416 384 442 -355 -4 -170 -371 416 -219 355 418 461 -290 -181 400 -94 215 152 -194 213 -258 140 -293 171 -67 -280 -321 -361 -415 -281 397 -345 258 -272 272 -370 
-313 -453 -230 -429 -93 17 -41 -346 -201 340 315 -316 -348 106 142 -238 -139 -237 -8 199 -490 -212 377 430 -425 292 370 87 -411 -254 376 -47 209 75 452 406 -205 -460 -439 -83 -504 -18 503 -417 169 500 242 376 -436 -391 -3 61 -500 103 258 135 -49 -188 338 355 311 -447 -48 -358 
136 480 22 -172 -509 -285 33 -71 -491 -253 -501 -42 448 504 -444 63 -453 438 -216 -226 -159 -434 451 367 347 99 -230 -346 -151 97 2 -364 505 -502 -200 503 194 -2 -153 -29 -218 -121 356 246 394 -147 367 497 -448 106 332 -355 -286 -284 323 259 -492 -362 -505 -209 -233 -239 458 -248 
-225 -448 -166 37 123 -201 -257 210 368 287 255 -266 96 303 336 363 276 -260 -111 76 -432 63 -129 -377 -307 -35 -405 227 -439 -330 96 -227 155 164 504 504 -287 373 -77 -468 -97 -323 -456 -273 -265 -109 418 109 268 400 390 111 482 84 303 124 452 -300 465 -82 284 128 354 -396 
-48 213 337 147 509 439 391 160 345 318 -442 30 359 56 -133 -499 102 488 -227 223 -239 -196 -334 -51 459 377 396 -214 241 382 -73 484 350 -433 -448 396 100 436 -73 -92 -35 -488 307 -502 253 -498 276 -48 199 383 261 -256 130 -342 342 -509 -166 -175 205 244 -346 287 -241 502 
475 468 -162 228 126 -171 275 -491 -349 420 129 186 -119 -113 403 -464 -234 276 -117 86 -365 -297 506 -380 -347 247 303 229 165 -467 -319 -23 208 -163 -210 -347 -90 41 129 -400 -462 -125 188 -125 128 217 -183 495 -321 -98 -226 421 434 43 -61 -346 245 -38 -385 91 194 149 -222 429 
-41 -102 47 -326 -2 170 199 -376 -502 -304 -412 -137 -346 -8 229 296 231 433 -333 -315 -92 129 -411 -189 -441 127 172 500 -290 -89 -300 -200 219 318 498 366 257 468 370 -416 97 -364 -332 400 142 -411 -406 -410 281 -223 327 -382 -2 -321 112 -348 -312 -206 -245 -247 -506 177 -43 95 
-233 314 367 83 -490 -138 -97 -336 -99 -271 69 -284 87 -471 -230 175 432 182 354 346 -219 342 -165 1 -71 -375 -187 361 -368 336 -436 506 -161 444 -24 21 -417 294 216 -127 -52 390 201 489 -428 408 -332 145 462 -431 291 449 -450 -489 -135 -438 -90 49 -483 492 234 -409 105 -498 
-245 14 63 -189 -483 355 -187 -306 48 59 420 406 -463 271 -98 -94 -95 -478 -295 11 114 308 188 17 -8 -65 -300 287 247 -466 423 78 61 -223 223 -404 492 140 479 -479 248 356 -323 -213 -130 480 -83 -346 -32 -414 323 -210 49 -140 -341 -134 -29 -411 -111 -58 -21 309 -7 283 
