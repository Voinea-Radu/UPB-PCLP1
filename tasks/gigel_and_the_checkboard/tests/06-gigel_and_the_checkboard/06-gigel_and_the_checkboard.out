7520
9 P
